app/controllers/planning_histories_controller.rb

```diff
@@ -10,4 +11,12 @@ def show
       format.html
     end
   end
+
+  private
+
+  def ensure_planning_history_is_enabled
+    return if @planning_application.planning_history_enabled?
+
+    render plain: "forbidden", status: :forbidden
```

Comment:
Is a 404 perhaps a more appropriate error response here?
As technically this page doesn't exist in production rather than it being forbidden

Reply:
I don't think so, cause the page is there, but is forbidden as we are disabling them. By having it 403, we can start doing AB testing!

For example once we have all the data for a specific local_authority we can enable this feature to that local_authority but not to others and so on